DataSources/Article.md

```diff
@@ -0,0 +1,167 @@
+# Analysis: Can a Cancer Patient Live a Normal Life?
+
+## Table of Contents
+1. [Abstract](#abstract)
+2. [How Different is Their Life from the Daily Life of an Ordinary Person?](#1-how-different-is-their-life-from-the-daily-life-of-an-ordinary-person)
+3. [How Much Extra Care and Attention They Needed](#2-how-much-extra-care-and-attention-they-needed)
+4. [Their Trauma, Anxiety, Depression, and Their Solutions, How They Can Keep Their Mind Fresh](#3-their-trauma-anxiety-depression-and-their-solutions-how-they-can-keep-their-mind-fresh)
+5. [Is Cancer Genetic?](#4-is-cancer-genetic)
+6. [Is Cancer Contagious?](#5-is-cancer-contagious)
+7. [Case Studies](#case-studies)
+8. [Conclusion](#conclusion)
+10. [References](#references)
+
+---
+
+## Abstract

```

Comment:
What is Cancer?
Cancer History
Origin
How cancer is formed?  (Include some picture)

https://academic.oup.com/book/27252/chapter-abstract/196863994?redirectedFrom=fulltext

Reply:
ok i will make the changes soon